Modules/xASL_module_ASL.m

```diff
@@ -346,7 +355,7 @@
 if ~isfield(x.Q, 'bUseBasilQuantification') || isempty(x.Q.bUseBasilQuantification)
 	x.Q.bUseBasilQuantification = false;
     
-    if x.modules.asl.bMultiPLD || x.modules.asl.bMultiTE
+    if x.modules.asl.bMultiPLD || x.modules.asl.bMultiTEQuantification
```

Comment:
See above, this will only work if you default bMultiTEQuantification to false regardless of having multiple TE, as default.

BTW: now we have a name discrepancy. bMultiPLD affects quantification so bMultiTE should also change ... quantification???

Reply:
As said above. if multiTE is missing in the data, bMultiTEQuantification is of course defaulted to false.

No need to change names -> as explained above.

Reply:
See above